build/package/Microsoft.DotNet.Cli.Nupkg.targets

```diff
@@ -11,46 +11,111 @@
     </PropertyGroup>
     
     <ItemGroup>
-      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils" >
+      <MicrosoftDotNetCliUtilsInputs Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils/**/*.cs;
```

Comment:
This stuff definitely falls in the "would be better if we were project based" category... :smile: